pidgin/app.py

```diff
@@ -130,7 +135,18 @@ def request_metadata(object_id):
         type file_name data_format file_size
         project_id object_id updated_datetime }} }}
         '''.format(file_type, object_id)
-    return send_query(query_txt)
+    data = send_query(query_txt)
+
+    # if this file does not have core_metadata_collections,
+    # only query the other metadata
+    if not data['data']:
+        simple_query_txt = '''{{ {} (object_id: "{}") {{
```

Comment:
This also seems like maybe a good candidate for a global constant (the `type`, `file_name`, etc. fields), which could be added to the query with
```python
' '.join(SIMPLE_QUERY_FIELDS)
```
(for example)

Reply:
Better yet, a function to issue the "standard" simple query would be a great way to go.